fvm/fvm_test.go

```diff
@@ -950,24 +986,22 @@ func TestBlockContext_ExecuteTransaction_InteractionLimitReached(t *testing.T) {
 				accounts, err := testutil.CreateAccounts(vm, view, programs, privateKeys, chain)
 				require.NoError(t, err)
 
-				_, txBody := testutil.CreateMultiAccountCreationTransaction(t, chain, 100)
+				_, txBody := testutil.CreateMultiAccountCreationTransaction(t, chain, 40)
 
 				txBody.SetProposalKey(chain.ServiceAddress(), 0, 0)
-				txBody.SetPayer(chain.ServiceAddress())
+				txBody.SetPayer(accounts[0])
 
-				err = testutil.SignPayload(txBody, accounts[0], privateKeys[0])
+				err = testutil.SignPayload(txBody, chain.ServiceAddress(), unittest.ServiceAccountPrivateKey)
 				require.NoError(t, err)
 
-				err = testutil.SignEnvelope(txBody, chain.ServiceAddress(), unittest.ServiceAccountPrivateKey)
+				err = testutil.SignEnvelope(txBody, accounts[0], privateKeys[0])
```

Comment:
I changed the order so its still authorized by service account but payer is not service account so it won't skip the limit